src/actions/ping/index.cr

```diff
@@ -0,0 +1,9 @@
+class Ping::Index < BrowserAction
+  include Auth::AllowGuests
+
+  get "/ping" do
+    return head 418 if Avram::Migrator::Runner.migrations.any?(&.new.pending?)
```

Comment:
@grepsedawk Can we use 503 instead of a silly status here?
```suggestion
    return head 503 if Avram::Migrator::Runner.migrations.any?(&.new.pending?)
```

Reply:
Oof, I'm sorry 😓 
The deploy was failing, and as the deploy failed from status check I copied this controller from a personal project.